lib/api/controllers/adminController.js

```diff
@@ -255,4 +256,61 @@ module.exports = function AdminController (kuzzle) {
       .then(response => kuzzle.pluginsManager.trigger('data:afterRefreshIndex', new ResponseObject(modifiedRequestObject, response)))
       .catch(err => q.reject(new ResponseObject(modifiedRequestObject || requestObject, err)));
   };
+
+  /**
+   * Gets the current autoRefresh value for the current index.
+   *
+   * @param {RequestObject} requestObject
+   * @returns {*}
+   */
+  this.getAutoRefresh = function (requestObject) {
+    var modifiedRequestObject = null;
+
+    return kuzzle.pluginsManager.trigger('data:beforeGetAutoRefresh', requestObject)
+      .then(newRequestObject => {
+        modifiedRequestObject = newRequestObject;
+
+        // even if being a get method, the autorefresh is set in the worker side
+        // we need to ask it the status
+        return kuzzle.workerListener.add(modifiedRequestObject);
+      })
+      .then(response => kuzzle.pluginsManager.trigger('data:afterGetAutoRefresh', new ResponseObject(modifiedRequestObject, response)))
+      .then(response => {
+        response.data.body = response.data.body.response;
+        return response;
+      })
+      .catch(err => q.reject(new ResponseObject(modifiedRequestObject || requestObject, err)));
```

Comment:
About rejecting a response object: you may want to synchronize with @ballinette: https://github.com/kuzzleio/kuzzle/pull/255